docs/build/build-storage.md

```diff
@@ -92,6 +92,17 @@ API that can be used to store a hash map with any arbitrary number of keys.
 These layered APIs act as runtime storage that allows you to store data in your blockchain. More information can 
 be found at the [`Runtime Storage Page`](https://docs.substrate.io/v3/runtime/storage/) on Substrate Developer Hub.
 
+### Crust Storage
+
```

Comment:
Still needs pretty: https://github.com/w3f/polkadot-wiki#formatting

Reply:
Still need to format the section so it is consistent with the style guide: 
![image](https://user-images.githubusercontent.com/25497083/151547067-5c469217-7d5d-4745-bfc0-1c56137b653f.png)